### PR TITLE
create docker-compose.yaml then configure the services and fix the"yarn dev" error 

### PR DESCRIPTION
**Pull request by iamCelott**
image

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    image: mariadb:latest
+    container_name: todo-db
+    environment:
+      MARIADB_ROOT_PASSWORD: S3cret!
+      MARIADB_DATABASE: todo
+      MARIADB_USER: todo
+      MARIADB_PASSWORD: todo
+    volumes:
+      - todo-db:/var/lib/mysql
+
```
Comment (pahrizal) [inline docker-compose.yaml:16]: Add `port` configuration section to bind host port 3307 to maridb container port 3306

**Review by pahrizal — COMMENTED** [review]
After the changes, please test this by running it using command: `docker compose up`.

File: docker-compose.yaml (hunk 1 above)
Comment (iamCelott) [reply docker-compose.yaml:16]: done sir.
image

**Review by pahrizal — APPROVED** [review]
Well done!

**Pull request merged**